### ISSUE TITLE
Docs: Better explanations of how things work

### ISSUE THREAD

**Issue by pboes** [litestar-org/litestar-fullstack#35]
Hi, I'm looking for a cross of Diango and FastAPI (read "Django building experience but lean and with first class pydantic and openapi support). Litestar and this repo in particular look like a great start but as somebody spoiled by Dingo abstracting away a lot of the backend, I'm wondering whether there can be a simple guide for how to use this repo coming from Django without having to learn all about SQLAlchemy etc. Or maybe you'd recommend a different stack? I assume many people looking to migrate from django to litestar will aim to do so via this repo. 

Thanks

---
> [!NOTE]  
> While we are open for sponsoring on [GitHub Sponsors](https://github.com/sponsors/litestar-org/) and 
> [OpenCollective](https://opencollective.com/litestar), we also utilize [Polar.sh](https://polar.sh/) to engage in pledge-based sponsorship.
>
> Check out all issues funded or available for funding [on our Polar.sh Litestar dashboard](https://polar.sh/litestar-org)
> * If you would like to see an issue prioritized, make a pledge towards it!
> * We receive the pledge once the issue is completed & verified
> * This, along with engagement in the community, helps us know which features are a priority to our users.

<a href="https://polar.sh/litestar-org/litestar-fullstack/issues/35">
<picture>
  <source media="(prefers-color-scheme: dark)" srcset="https://polar.sh/api/github/litestar-org/litestar-fullstack/issues/35/pledge.svg?darkmode=1">
  <img alt="Fund with Polar" src="https://polar.sh/api/github/litestar-org/litestar-fullstack/issues/35/pledge.svg">
</picture>
</a>

**Comment by v3ss0n** [comment]
i will help with this this weekend.

**Comment by nathanielarking** [comment]
I'm in the same spot here! Migrating from Django to Litestar and while the Litestar docs are great at the moment, there isn't a lot on how to structure an application. This repo is helpful so far but I would find it more helpful if there was more explanation, so we can understand why things are being done the way they are.

Articles like [this one](https://simpleisbetterthancomplex.com/tutorial/2021/06/27/how-to-start-a-production-ready-django-project.html) were very helpful to me learning Django in the first place.

**Comment by dkout** [comment]
+1 here. Not necessarily Django migration, but an article explaining how things work and why certain choices were made would be very useful. Thanks!

**Comment by AgarwalPragy** [comment]
> +1 here. Not necessarily Django migration, but an article explaining how things work and why certain choices were made would be very useful. Thanks!

Absolutely! An article explaining the directory structure, why certain choices were made, and how to make changes would be amazing!

Funding this issue with 100$

**Comment by peterschutt** [comment]
I can confirm we can see this in polar. Watch the issue for updates - thanks for your pledge!
